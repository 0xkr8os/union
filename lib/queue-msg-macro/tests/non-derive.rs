use queue_msg_macro::msg_struct;

#[msg_struct]
fn function() {}

fn main() {}
