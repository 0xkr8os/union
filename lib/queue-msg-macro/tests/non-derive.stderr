error: expected one of: `struct`, `enum`, `union`
 --> tests/non-derive.rs:4:1
  |
4 | fn function() {}
  | ^^
